src/form/MapSelectAssetOrPoint.tsx

```diff
@@ -0,0 +1,246 @@
+// Todo: This component should probably also be used for the raster alarms
```

Comment:
It would be nice if this component can also be reused by the RasterPointSelection field in the Raster Alarms form. So the next step, we can reuse both the GeometryField and this MapSelectAssetOrPoint field for the RasterAlarmForm. I will make a separate ticket for this task.